src/plugins/list/commands.js

```diff
@@ -14,6 +16,6 @@ export function changeListType(change, type) {
     if (node === null) {
         change.wrapInList(type)
     } else {
-        change.setNodeByKey(node.key, type)
```

Comment:
Only properties related to list's type should be cleared. Aside from `class`, `mark-prefix`, `mark-suffix`, `item-sep`, and `type` should also be preserved.